deploy/staging/deployment.yaml

```diff
@@ -0,0 +1,589 @@
+                Wormwood Scrubs
+            - name: SMTP_DOMAIN
+              value: "mailtrap.io"
+            - name: SMTP_HOSTNAME
+              value: "smtp.mailtrap.io"
+            - name: SMTP_PORT
+              value: "465"
+            - name: SMTP_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: smtp_username
+            - name: SMTP_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: smtp_password
+            - name: REMOVE_LOAD_TEST_DATA
+              value: "true"
+            - name: PUBLIC_SERVICE_URL
+              value: "https://public-staging-pvb.dsd.io"
+            - name: STAFF_SERVICE_URL
+              value: "https://staff-staging-pvb.dsd.io"
+            - name: STAFF_SERVICE_URL
+              value: "http://prison-visits-staff-info.s3-website-eu-west-1.amazonaws.com"
+            - name: PRISON_ESTATE_IPS
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: prison_estate_ips
+            - name: SENTRY_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: sentry_dsn
+            - name: SENTRY_JS_DSN
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: sentry_js_dsn
+            - name: ZENDESK_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: zendesk_token
+            - name: ZENDESK_URL
+              value: ""
+            - name: GA_TRACKING_ID
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: ga_tracking_id
+            - name: PVB_TEAM_EMAIL
+              value: "pvb-team@digital.justice.gov.uk"
+            - name: RACK_ENV
+              value: "production"
+            - name: SECRET_KEY_BASE
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: secret_key_base
+            - name: RAILS_ENV
+              value: "production"
+            - name: RAILS_SERVE_STATIC_FILES
+              value: "true"
+            - name: WEB_CONCURRENCY
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: rails_web_concurrency
+            - name: REDIS_URL
+              valueFrom:
+                secretKeyRef:
+                  name: prison-visits-booking-staff-secrets
+                  key: redis_url
+        - name: prison-visits-booking-staff-sidekiq
```

Comment:
Need to refactor deployment script to use config maps in order to remove duplicated env vars